### PR TITLE
Update to 2021.2.50-3 version

### PR DESCRIPTION
**Pull request by rinatdobr**

### CODE REVIEW

**Comment by mkosola** [comment]
Please update also ca-certificates.changes according to new version and changes.

**Comment by rinatdobr** [comment]
> Please update also ca-certificates.changes according to new version and changes.

Done.

**Review by okodron — APPROVED** [review]

File: ca-certificates.spec [hunk 1]
```diff
@@ -244,9 +246,15 @@ chmod 444 $RPM_BUILD_ROOT%{catrustdir}/extracted/%{java_bundle}
 touch $RPM_BUILD_ROOT%{catrustdir}/extracted/edk2/cacerts.bin
 chmod 444 $RPM_BUILD_ROOT%{catrustdir}/extracted/edk2/cacerts.bin
 
-# /etc/ssl/certs symlink for 3rd-party tools
-ln -s ../pki/tls/certs \
+# /etc/ssl symlinks for 3rd-party tools and cross-distro compatibility
+ln -s /etc/pki/tls/certs \
```
Comment (xfade) [inline ca-certificates.spec:250]: Use %{_sysconfdir} instead of /etc ?

Comment (pvuorela) [reply ca-certificates.spec:250]: That looks like upstream change. I wouldn't create delta unless there's a clear need.

File: ca-certificates.changes [hunk 2]
```diff
@@ -1,3 +1,6 @@
+* Wed Oct 13 2021 Rinat Dobrokhotov <r.dobrokhotov@omprussia.ru> - 2021.2.50-3
```
Comment (pvuorela) [inline ca-certificates.changes:1]: Release 4 in spec? Upstream with 3.

File: ca-certificates.spec (hunk 1 above)
Comment (rinatdobr) [reply ca-certificates.spec:250]: @pvuorela , I agree.

File: ca-certificates.changes (hunk 2 above)
Comment (rinatdobr) [reply ca-certificates.changes:1]: Fixed.

File: ca-certificates.spec (hunk 1 above)
Comment (xfade) [reply ca-certificates.spec:250]: Yes. Let's not complicate things then.

**Comment by pvuorela** [comment]
LGTM

**Pull request merged**